core/src/main/java/com/homeaway/streamplatform/streamregistry/StreamRegistryApplication.java

```diff
@@ -77,6 +70,12 @@
 import com.homeaway.streamplatform.streamregistry.resource.ClusterResource;
 import com.homeaway.streamplatform.streamregistry.resource.RegionResource;
 import com.homeaway.streamplatform.streamregistry.resource.StreamResource;
+import com.homeaway.streamplatform.streamregistry.service.ClusterService;
+import com.homeaway.streamplatform.streamregistry.service.RegionService;
+import com.homeaway.streamplatform.streamregistry.service.StreamClientService;
+import com.homeaway.streamplatform.streamregistry.service.StreamService;
+import com.homeaway.streamplatform.streamregistry.service.impl.*;
+import com.homeaway.streamplatform.streamregistry.service.impl.ClusterServiceImpl;
```

Comment:
we need to standardize on an import order. (#17 is related)

Reply:
Please remove the .* imports from your IDE that should only be used for statics and only in tests.

Reply:
fixed it.